action.yml

```diff
@@ -95,7 +103,9 @@ runs:
       shell: bash
       id: sync
       env:
-        GITHUB_TOKEN: ${{ inputs.github_token }}
+        SOURCE_GH_TOKEN: ${{ inputs.source_gh_token}}
+        TARGET_GH_TOKEN: ${{ inputs.target_gh_token }}        
+        GITHUB_TOKEN: ${{ inputs.github_token }}        
```

Comment:
Hmm this part I do not fully get @alexvanderberkel do we need the env variable `GITHUB_TOKEN`?

Reply:
I think I kind of wanted to keep the default token of the pipeline this way